components/Tabs/Tabs.js

```diff
@@ -2,24 +2,29 @@
 class TabsItem {
   constructor(element) {
     // attach dom element to object. Example in Tabs class
+    this.element = element;
   }
 
   select() {
     // should use classList
+    this.element.classList.add("Tabs__item-selected");
   }
 
   deselect() {
     // should use classList
+    this.element.classList.remove(".Tabs__item-selected");
   }
 }
 
 class TabsLink {
   constructor(element, parent) {
-    this.element;// attach dom element to object
-    this.tabs;// attach parent to object
-    this.tabsItem;// assign this to the associated tab using the parent's "getTab" method by passing it the correct data
+    this.element = element;// attach dom element to object
+    this.tabs = parent;// attach parent to object
+    this.tabsItem = parent.getTab(this.element.dataset.tab);// assign this to the associated tab using the parent's "getTab" method by passing it the correct data
     // reassign this.tabsItem to be a new instance of TabsItem, passing it this.tabsItem
+    this.tabsItem = new TabsItem(this.tabsItem);
     this.element.addEventListener('click', () => {
```

Comment:
Your tab buttons seem to be working properly to a certain extent. I would take a closer look at how you are calling them! What is happening on my computer is that instead of switching boxes, they are just adding to the side.